CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+CHANGES IN VERSION 1.14.10 (FROM 1.13.0)
+===================================
+
+ Appium 1.14.0 is a minor release.
+
+ #### General
+* Minimum Node version is bumped up to v10
+* The default driver for Android is now set to [UiAutomator2](https://github.com/appium/appium-uiautomator2-driver). If the [UiAutomator1](https://github.com/appium/appium-android-driver) driver is desired, then this can be achived by setting `automationName=UiAutomator1` in the capabilities
+
+ #### Android (Espresso)
+* Fix problems with activity startup not working when the package name value in `appActivity` is different from the one in `appPackage` [#441](https://github.com/appium/appium-espresso-driver/pull/441)
```

Comment:
Add a mobile helper to disable autofill dialog in Android O (#456)[https://github.com/appium/appium-espresso-driver/pull/456]

Reply:
Done